### ISSUE TITLE
AccessibilityInfo conflict?

### ISSUE THREAD

**Issue by JeremyBradshaw7** [christopherdro/react-native-html-to-pdf#114]
Introducing this package to a RN 0.55 project causes build failures on Android:

error: bundling failed: ambiguous resolution: module `/Users/jeremybradshaw/ccf/node_modules/react-native/Libraries/react-native/react-native-implementation.js` tries to require `AccessibilityInfo`, but there are several files providing this module. You can delete or fix them: 

  * `/Users/jeremybradshaw/ccf/node_modules/react-native-html-to-pdf/example/node_modules/react-native/Libraries/Components/AccessibilityInfo/AccessibilityInfo.android.js`
  * `/Users/jeremybradshaw/ccf/node_modules/react-native/Libraries/Components/AccessibilityInfo/AccessibilityInfo.android.js`

A similar failure occurs on iOS: UnableToResolveError: Unable to resolve module `AccessibilityInfo`

**Comment by JeremyBradshaw7** [comment]
Can I just delete the node_modules/react-native-html-to-pdf/example folder? Why is that causing build issues?

**Comment by ylac** [comment]
Yep that worked for me. Had to restart the packager too.

**Comment by ylac** [comment]
Think it's because the example/package.json specifies an old react-native version.

**Comment by Esemesek** [comment]
Issue can also be solved by adding ignore patterns to metro bundler. https://facebook.github.io/metro/docs/en/configuration#blacklistre
